components/countdown/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+
+import Box from '@components/box';
+
+const getTargetDate = (): Date => {
+	const now = new Date();
+	const sundayNumber = 0;
+	const timezone = 'America/New_York';
+	const today = new Date(now);
+	today.setMilliseconds(0);
+	today.setSeconds(0);
+	today.setMinutes(0);
+	today.setHours(0);
+
+	const nextSunday = new Date(today);
+
+	do {
+		nextSunday.setDate(nextSunday.getDate() + 1);
+	} while (nextSunday.getDay() !== sundayNumber);
+
+	return new Date(nextSunday.toLocaleString('en-US', { timeZone: timezone }));
```

Comment:
This function is a little hard to follow. It could use comments or more descriptive variable names. 

And there could be some improvement on the locality of the variables, for eg:

```suggestion
	const now = new Date();
	const today = new Date(now);
	
	today.setMilliseconds(0);
	today.setSeconds(0);
	today.setMinutes(0);
	today.setHours(0);

	const nextSunday = new Date(today);

	const sundayNumber = 0;
	do {
		nextSunday.setDate(nextSunday.getDate() + 1);
	} while (nextSunday.getDay() !== sundayNumber);


	const timezone = 'America/New_York';
	return new Date(nextSunday.toLocaleString('en-US', { timeZone: timezone }));
```

Reply:
Great readability suggestion!
Applied at https://github.com/ardriveapp/inferno-leaderboard/pull/2/commits/805e1f9541a9050fc53f84aa5c6f2d61d083ef85